### PR TITLE
Fixes #55: python_wrapper_helper uses CMAKE_CXX_STANDARD

### LINKED ISSUE #55: Use standard from CMake
Exactly like reported here:

* https://github.com/intel/ad-rss-lib/issues/101

python_wrapper_helper.py should use CMAKE_CXX_STANDARD to pass gccxml the proper C++ flavor to use.

### PR DESCRIPTION
**Pull request by daedric**
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/map/56)

### CODE REVIEW

**Review by berndgassmann — DISMISSED** [review]
> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/map/56)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @daedric)

**Review by daedric — COMMENTED** [review]
> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/map/56)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @daedric)

**Review by daedric — COMMENTED** [review]
> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/map/56)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @daedric)

File: cmake/python/python_wrapper_helper.py.in [hunk 1]
```diff
@@ -79,7 +79,7 @@ def generate_python_wrapper(header_directories, include_paths, library_name, cpp
         start_with_declarations=declarations)
 
     # Set include dirs and cflags to avoid warnings and errors
-    xml_generator_config.append_cflags("-std=c++11")
+    xml_generator_config.append_cflags("-std=c++@CMAKE_CXX_STANDARD@")
```
Comment (berndgassmann) [inline cmake/python/python_wrapper_helper.py.in:83]: Looks like older versions of compiler/OS have problems with some warnings on cast_xml calls; maybe that was the reason why we didn't update the flags originally.
I tested a bit around locally (ok, only have Ubuntu20.04 where the error isn't occurring), but maybe adding a warning suppression for the parser helps
xml_generator_config.append_cflags("-w")
or less restrictive and directly addressing the spdlog warning explicitly making the problem
xml_generator_config.append_cflags("-Wno-error=invalid-constexpr")

**Review by berndgassmann — CHANGES_REQUESTED** [review]

File: cmake/python/python_wrapper_helper.py.in (hunk 1 above)
Comment (berndgassmann) [reply cmake/python/python_wrapper_helper.py.in:84]: just came to my mind, that I could try it be editing the file online.
Let's see what the CI makes out of it.
I LOVE multiple compilers on multiple systems :-)

**Review by berndgassmann — APPROVED** [review]

**Review by berndgassmann — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/map/56)* status: all files reviewed, 1 unresolved discussion (waiting on @daedric)

**Review by berndgassmann — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/carla-simulator/map/56)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @daedric)

**Pull request merged**